examples/shared_samples/fc_network.rb

```diff
@@ -32,6 +40,15 @@
 # api_version = 500 & variant = Synergy to OneviewSDK::API500::C7000::FCNetwork
 # api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::FCNetwork
 # api_version = 600 & variant = Synergy to OneviewSDK::API600::C7000::FCNetwork
+# api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::FCNetwork
+# api_version = 800 & variant = Synergy to OneviewSDK::API800::C7000::FCNetwork
+# api_version = 1000 & variant = C7000 to OneviewSDK::API1000::C7000::FCNetwork
+# api_version = 1000 & variant = Synergy to OneviewSDK::API1000::C7000::FCNetwork
+# api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::FCNetwork
+# api_version = 1200 & variant = Synergy to OneviewSDK::API1200::C7000::FCNetwork
+# api_version = 1600 & variant = C7000 to OneviewSDK::API1600::C7000::FCNetwork
+# api_version = 1600 & variant = Synergy to OneviewSDK::API1600::C7000::FCNetwork
```

Comment:
Need to correct the class inheritance for all the api version with Synergy variant mention in this file from line 36 till 50.

Reply:
Updated as Synergy variant